packages/modeling/src/geometries/geom2/type.d.ts

```diff
@@ -0,0 +1,9 @@
+import Vec2 from '../../maths/vec2/type'
```

Comment:
sure, but i think it's best to split up the definitions in multiple files for the same reasons it's best not to have all your code in a single file. as a specific reason, with the definitions being 1-to-1 with their respective code files, it means you can import at any level and the types will work, whether you import `@jscad/modeling` or `@jscad/modeling/src/maths` it will work. this is better for me because i've been importing `@jscad/modeling/src/maths` and don't want to import things i won't use (and unnecessarily increase bundle size, at least until i've tested this and seen if tree-shaking might work).

Reply:
Sounds like a good reason to keep the current organization of files.